### ISSUE TITLE
Pokemon Heartgold White Screens after soft reset

### ISSUE THREAD

**Issue by kellyGipson** [DS-Homebrew/nds-bootstrap#1428]

**Comment by RocketRobz** [comment]
In which point did you soft-reset at?

**Comment by kellyGipson** [comment]
I have tried soft resetting from the starter selection area and from the title screen. Both have the same result

**Comment by RocketRobz** [comment]
I use an R4 Ultra, and soft-resetting works fine on my side. Maybe it's a flashcard/DLDI-specific issue.

**Comment by kellyGipson** [comment]
Are there any recommended steps I might be able to take to mitigate this issue? Or perhaps some part of code I can look into that's perhaps related to the issue?

**Comment by RocketRobz** [comment]
Check if `pagefile.sys` exists in `sd:/_nds/`. If it does exist, how big is it?

**Comment by kellyGipson** [comment]
It does exist and it is exactly 4.00MB(4,194,304 bytes)

**Comment by RocketRobz** [comment]
Are you able to open the in-game menu by pressing L+Down+Select?

**Comment by kellyGipson** [comment]
At what point should I be able to enter this menu?

**Comment by RocketRobz** [comment]
Yes, that's what I meant. See if the `Reset Game` option works for you.

**Comment by kellyGipson** [comment]
Reset Game causes the game to freeze

**Comment by RocketRobz** [comment]
Hmm, is the SD card formatted with 32kb cluster size?

**Comment by kellyGipson** [comment]
I'll do a fresh format to make sure. I'm not sure what the cluster size is

**Comment by kellyGipson** [comment]
The cluster size is 32kb

**Comment by WildGardevoir** [comment]
I have found a solution for this issue.

I am using autoboot Twilightmenu on my ace3ds+ card, and the way I fixed soft resetting was by:

Starting the game normally

After the game loads in, I open the twilightmenu in game by pressing L+Down+Select

Then click "Quit game", it will take you back to the Twilightmenu home, from there open the game again and soft resetting will work like normal.

**Comment by Gokazary** [comment]
I have the same problem of white screen on soft reset but only with Pokemon Diamond and Pearl. It works fine with all my other games. I tried to quit game with using Twilight Menu option, but soft reset after this still produce a white screen.
